horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/RunServiceImpl.java

```diff
@@ -374,10 +342,9 @@ public void updateAccess(int id, String owner, Access access) {
 
     @PermitAll // all because of possible token-based upload
```

Comment:
I think we need to restrict this to `@RolesAllowed(Roles.UPLOADER)` to limit the to users with that role